src/main/java/fi/csc/chipster/jobhistory/JobHistoryService.java

```diff
@@ -181,7 +181,8 @@ private void saveJobHistory(Job job){
 		jobHistory.setEndTime(job.getEndTime());
 		jobHistory.setOutput(job.getScreenOutput());
 		jobHistory.setJobStatus(job.getStateDetail());
-		
+		jobHistory.setUserName(job.getCreatedBy());
+		System.out.println("saveJobHistory() " + job.getCreatedBy());
```

Comment:
You may have already changed this already, but just in case: System.out.printlin is fine during development, but should be replaced with logger or removed in the end.